client/src/components/home/SeriesCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import WishlistIcon from './WishlistIcon';
+import {
+  MDBCard,
+  MDBCardBody,
+  MDBCardTitle,
+  MDBCol,
+  MDBCardImage
+} from 'mdb-react-ui-kit';
+
+export default function SeriesCard(props) {
```

Comment:
You can do in the first line:
`const { series } = props`
and then use `series` instead of `props.series` everywhere in this file